abaco.go

```diff
@@ -38,9 +38,9 @@ func gIndex(p *packets.Packet) GroupIndex {
 
 // FrameTimingCorrespondence tracks how we convert timestamps to FrameIndex
 type FrameTimingCorrepondence struct {
-	CountsPerRow  uint64
-	LastTimestamp packets.PacketTimestamp
-	LastRowCount  FrameIndex
+	SubframeRate      uint64 // Ratio of timestamp rate to subframe rate
```

Comment:
We should be consistent about `SubframeRate` vs `SubframeDivisions` vs `SubframeRatio`. Pick one and never use another. `SubframeTicks` is also reasonable.

Reply:
I like `SubframeRate` the most after having gone most of the way through this review.

Reply:
Hmm, I was of the same mind at first. Then I decided that "rate" implies units of inverse seconds. This is a unitless quantity that divides the frame. Ratio is reasonable, but Divisions is what I settled on. I'll use it here.

Reply:
I think `SubframeRate` makes sense because the rate has units inverse frames.